Components/Optimizers/AdaptiveStochasticVarianceReducedGradient/elxAdaptiveStochasticVarianceReducedGradient.hxx

```diff
@@ -644,7 +644,7 @@ AdaptiveStochasticVarianceReducedGradient<TElastix>
     }
 
     this->m_CurrentInnerIteration = 0;
-    const int gradientFactor = 0.8;
```

Comment:
Good catch @haampie Still wondering what this `int gradientFactor = 0.8` was originally meant for... @mstaring do you have a clue?

Reply:
looks good. No, just a research code